src/main/resources/assets/create/lang/es_es.json

```diff
@@ -51,7 +51,7 @@
 	"block.create.chiseled_scoria": "Escoria cincelada",
 	"block.create.chiseled_weathered_limestone": "Piedra caliza erosionada cincelada",
 	"block.create.chocolate": "Chocolate",
-	"block.create.chute": "Ducto",
+	"block.create.chute": "Vertedor",
```

Comment:
Looking at the object itself it is not a spillway, if you look at it, it is a conduit through which things fall.